src/components/qualitySelector/qualitySelector.js

```diff
@@ -98,6 +124,18 @@ const qualitySelector = {
     if (sourceMenuButton) {
       const qualityLevels = player.qualityLevels();
       if (qualityLevels && qualityLevels.length > 1) {
+        let levels = qualityLevels.levels_.filter((q) => q.enabled);
+        if (levels.length === 1) {
+          let idx = qualityLevels.levels_.findIndex(l => l.id === levels[0].id);
+          sourceMenuButton.children()[1].children()[idx].selected(true);
```

Comment:
@yaniv-li following our previous discussion 
for marking selected option, see [what they did here](https://github.com/cloudinary/cloudinary-video-player/pull/216/files#diff-3bba335ded3dcda3c175fc8e6c2dd199bbaba7641b11ca9dd014ec0a2090c131R18):
```
let levels = this.player().qualityLevels();
levels[i].enabled = true;
```

Reply:
it should be enabled this is just to make sure the visual part is ok